poker-hand-evaluator/src/evaluator/HandEvaluatorTest.java

```diff
@@ -56,19 +56,19 @@ void shouldEvaluateTwoTwoPairs() throws InvalidHandSizeException {
     @Test
     void shouldEvaluateTwoHighCards() throws InvalidHandSizeException {
         Hand firstHand = new Hand(List.of(
-            new Card(CardColour.CLUBS, CardValue.ACE),
-            new Card(CardColour.SPADES, CardValue.KING),
-            new Card(CardColour.HEARTS, CardValue.QUEEN),
-            new Card(CardColour.CLUBS, CardValue.JACK),
-            new Card(CardColour.DIAMONDS, CardValue.THREE))
+                new Card(CardColour.CLUBS, CardValue.ACE),
+                new Card(CardColour.SPADES, CardValue.KING),
+                new Card(CardColour.HEARTS, CardValue.QUEEN),
+                new Card(CardColour.CLUBS, CardValue.JACK),
+                new Card(CardColour.DIAMONDS, CardValue.THREE))
         );
 
         Hand secondHand = new Hand(List.of(
-            new Card(CardColour.CLUBS, CardValue.TWO),
-            new Card(CardColour.SPADES, CardValue.ACE),
-            new Card(CardColour.HEARTS, CardValue.KING),
-            new Card(CardColour.CLUBS, CardValue.JACK),
-            new Card(CardColour.DIAMONDS, CardValue.QUEEN))
+                new Card(CardColour.CLUBS, CardValue.TWO),
+                new Card(CardColour.SPADES, CardValue.ACE),
+                new Card(CardColour.HEARTS, CardValue.KING),
+                new Card(CardColour.CLUBS, CardValue.JACK),
+                new Card(CardColour.DIAMONDS, CardValue.QUEEN))
```

Comment:
You are testing 5-card hands' comparison, but you are not testing if the best hand gets selected from the 21 options.